custom_connectors/custom_auth/eventbrite_connector.rb

```diff
@@ -0,0 +1,179 @@
+{
+  title: "Eventbrite (custom)",
+
+  connection: {
+    fields: [
+      {
+        name: "personal_token",
+        control_type: "password",
+        hint: "Find your personal token here: https://www.eventbrite.com/myaccount/apps/"
+        optional: false
+      }
+    ],
+
+    authorization: {
+      type: "custom_auth",
+
+      apply: lambda do |connection|
+        headers("Authorization": "Bearer #{connection["personal_token"]}")
+      end
+    },
+
+    base_uri: lambda do
+      "https://www.eventbriteapi.com"
+    end
+  },
+
+  test: lambda do
+    get("/v3/users/me")
+  end,
+
+  actions: {
+    get_sales_report: {
+      execute: lambda do |connection, input|
+        get("/v3/reports/sales", input)
+      end,
+
+      input_fields: lambda do |object_definition|
+        [
+          {
+            name: "event_status",
+            control_type: "select",
+            pick_list: [
+              ["All", "all"],
+              ["Live", "live"],
+              ["Ended", "ended"]
+            ],
+            optional: false
+          }
+        ]
+      end,
+
+      output_fields: lambda do |object_definition|
+        object_definition["sales_report"]
+      end
+    }
+  },
+
+  object_definitions: {
+    sales_report: {
+      fields: lambda do
+        [
+          {
```

Comment:
Make all of this single line. No need for attributes that can be defaulted.
{ "name": "timezone" }